### DISCUSSION TITLE
Access C# code from C++

### DISCUSSION THREAD

**Discussion by sacOO7** [nxrighthere/UnrealCLR#271] [Help] [answered]
Can I access unrealCLR C# code from C++

**Comment by nxrighthere** [comment, marked as answer]
It's possible by using blueprints or by writing a custom C++ layer (see how [`UnrealCLR::ManagedCommand()`](https://github.com/nxrighthere/UnrealCLR/blob/8a517e23b3b76399d77d50a3b8ad0cb8887c2a8e/Source/Managed/Runtime/Runtime.cs#L145) is implemented).
